docs/getting-started/work-packages-introduction/README.md

```diff
@@ -42,19 +42,19 @@ In the empty form on the right, you can enter all relevant information for this
 
 Click the green **Save** button to create the work package.
 
-![split-screen-work-packages](split-screen-work-packages.png)
+![split-screen-workpackages](split-screen-workpackages.png)
 
 The work package will the be displayed in the list view:
 
-![list-view-work-package](1569611758166.png)
+ ![create-work-package-define-project](create-work-package-define-project-6669224.png)
 
 Another option to create a work package is to do it from the header menu. The [work package types](../../user-guide/projects/project-settings/work-package-types/#work-package-types) that are activated, will be shown and you can select the relevant work package type to be created.
 
 ![create-work-package-header](create-work-package-header.png)
 
 Once you click on the work package type that you want to create, the work package detail view opens and you have to **select the project** that you want to create the work package for. 
 
-![create-work-package-define-project](create-work-package-define-project.png)
+![Bildschirmfoto 2022-03-07 um 17.25.47](Bildschirmfoto 2022-03-07 um 17.25.47.png)
```

Comment:
please rename the image file and change the name here, so there are no spaces in it.